x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -179,9 +179,9 @@ CYPRESS_BASE_URL=http(s)://<username>:<password>@<kbnUrl> CYPRESS_ELASTICSEARCH_
 
 ## Best Practices
 
-### Clean up the state between tests
+### Clean up the state 
 
-Remember to clean up the state of the test after its execution.
+Remember to use the `cleanKibana` method before starting the execution of the test
```

Comment:
Rather than leaving this as a task for the test writer, we could enforce it as a global `beforeEach` hook.

Reply:
Here I'm not 100% sure. There are some cases where using before instead of beforeEach, save us execution time. So we need to take into consideration that if we move that to a global hook, we are going to make our tests more slow. Another reason is because I don't like how Cypress handles right now global hooks. 

Anyway, this is a think we can investigate/work in a different PR :)